connector/sumconnector/config.go

```diff
@@ -3,6 +3,17 @@
 
 package sumconnector // import "github.com/open-telemetry/opentelemetry-collector-contrib/connector/sumconnector"
 
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/component"
+	"go.uber.org/multierr"
```

Comment:
We should use `errors.Join` rather than multierr. Context: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/25121